cmd/operator/main.go

```diff
@@ -8,18 +8,13 @@ import (
 	"os"
 	"time"
 
-	"github.com/google/uuid"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 
 	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/client-go/kubernetes"
```

Comment:
If we're going to clean up this package, can we also remove those two `_` imports that appear to serve no purpose? (FWIW, they're also in the manager package too...)

Reply:
Taking out magic `_` imports scares me, as they _sometimes_ seem to have implications for code generation and other not-runtime ballyhoo. Can I do that in a separate PR?